uportal-war/src/main/webapp/media/skins/common/javascript/uportal/entity-selector.js

```diff
@@ -495,7 +495,7 @@ var up = up || {};
         // Cache.
         closeSearch = that.locate("closeSearch");
         searchForm = that.locate("searchForm");
-        searchField = searchForm.find("input[type=text]");
+        searchField = searchForm.find("input[type=search]");
```

Comment:
Based on other changes, you might want to adjust the way the `searchField` control is used...

```
        // Apply.
        searchField.val(that.options.messages.searchValue);
        searchDropDown.css({'top': searchField.outerHeight()});

        // Binds 'click' & 'focus' events to input field.
        searchField.bind("click focus", function () {
            $(this).val("");
        });//end:listener.
```